.coveralls.yml

```diff
@@ -0,0 +1,3 @@
+src_dir: .
```

Comment:
If that's the case, I'd also like to get rid of the `.semver` file we seem to add everywhere.